introduction.md

```diff
@@ -4,4 +4,7 @@ Make a local clone of your fork of the "dojo_rules" repository. Make sure you do
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?